WordPress/WordPressNotificationServiceExtension/Sources/NotificationService.swift

```diff
@@ -24,6 +24,18 @@ class NotificationService: UNNotificationServiceExtension {
     // MARK: UNNotificationServiceExtension
 
     override func didReceive(_ request: UNNotificationRequest, withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void) {
+        UserDefaults(suiteName: WPAppGroupName)?.synchronize()
+        if shouldFilterNotification() {
+            // TODO
+            /// Once com.apple.developer.usernotifications.filtering we can pass empty content to silence notifications
+            /// https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_usernotifications_filtering
+            let content = UNMutableNotificationContent()
+            content.title = "Filtered"
+            content.body =  "This notication won't appear"
+            contentHandler(content)
```

Comment:
⚠️ Once entitlement is enabled, we can call:

```
contentHandler(UNNotificationContent())
```

to skip showing notification